lib/workers/repository/process/write.ts

```diff
@@ -156,7 +156,12 @@ export async function writeUpdates(
       branchState,
       commitFingerprint,
     );
-    const res = await processBranch(branch);
+    // no need to include the env object in the fingerprint;
+    // the SHA changes when the user modifies the config file, and we'll update the branch regardless
+    const res = await processBranch({
+      ...branch,
+      userConfiguredEnv: config.env,
+    });
```

Comment:
Can you describe more about this? I thought env would be part of the fingerprint as it can change the outcome

Reply:
sha can stay same if the env is inside a preset. so we need to consider it